src/mediapipe_internal/mediapipegraphdefinition.cpp

```diff
@@ -422,7 +422,7 @@ Status MediapipeGraphDefinition::initializeNodes() {
             }
 
             std::shared_ptr<PythonNodeResources> nodeResources = nullptr;
-            Status status = PythonNodeResources::createPythonNodeResources(nodeResources, config.node(i), pythonBackend);
+            Status status = PythonNodeResources::createPythonNodeResources(nodeResources, config.node(i), pythonBackend, mgconfig.getBasePath());
```

Comment:
Is the base_path an absolute path to the directory with config.json or graph.pbtxt?

Reply:
mgconfig.getBasePath() return path to graph.pbtxt